modules/tensor_mechanics/tests/capped_mohr_coulomb/small_deform9_cosserat.i

```diff
@@ -0,0 +1,488 @@
+# Using Cosserat with large layer thickness, so this should reduce to standard
+# Using CappedMohrCoulombCosserat with tensile failure only
+# A single unit element is stretched in a complicated way that
+# the trial stress is
+#        1.51515             0.8        0.666667
+#            0.8        -3.74545    -1.85037e-17
+#            0.7    -1.66533e-17        -1.27273
+# with symmetric part
+#        1.51515             0.8        0.683333
+#            0.8        -3.74545    -1.85037e-17
+#            0.683  -1.66533e-17        -1.27273
```

Comment:
different number of digits in \sigma_{sym-31} and \sigma_{sym-13}; is this intentional?